keyboards/thevankeyboards/minivan/keymaps/lexworth/rules.mk

```diff
@@ -0,0 +1,21 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in 
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
```

Comment:
For future reference, the only rules that are *necessary* to include in a keymap-level `rules.mk` file such as this are rules that differ from those set at the keyboard level (`keyboards/thevankeyboards/minivan/rules.mk`).

For you, that would be:

```
CONSOLE_ENABLE = no         # Console for debug(+400)
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
```